gp_fit_real_data/fit_uv_real_data.py

```diff
@@ -84,10 +77,10 @@ def objective_closure():
 
             fixed_noise = np.mean(np.abs(uv_band_flux/snr))
             set_trainable(m.likelihood.variance, False)  # We don't want to optimise the noise level in this case.
-            m.likelihood.variance = fixed_noise
+            m.likelihood.variance.assign(fixed_noise)
 
         opt = gpflow.optimizers.Scipy()
-        opt.minimize(objective_closure, m.trainable_variables, options=dict(maxiter=100))
+        opt.minimize(m.training_loss, m.trainable_variables, options=dict(maxiter=1000))
```

Comment:
No need for always writing your objective functions:) (SVGP models have model.training_loss_closure(data) instead)
Also, I'd be worried 100 iterations is a bit on the short side and not actually converged yet. In fact, I'd probably leave out the maxiter setting altogether...
```suggestion
        opt.minimize(m.training_loss, m.trainable_variables)
```

Reply:
Thanks for the explanation! Makes sense to set the default `maxiter` value higher than 100 in the general case! (I just checked with 1000 and this gives a minuscule improvement in the 6th decimal of the log marginal likelihood!). I'm probably missing something obvious but I just followed up GPflow's scipy optimiser documentation in PyCharm and couldn't find much information on default values for `maxiter`. I don't suppose you know what the default behaviour is?

Reply:
All extra options are simply passed through to Scipy's minimize. I believe not passing maxiter means "infinite", but I'm not 100% certain on that.

Reply:
PS- if you could also update your medium blog post to use model.training_loss instead of writing a custom objective, that'd be great : )

Reply:
Done!